tests/baselines/reference/TupleType5.errors.txt

```diff
@@ -1,7 +0,0 @@
-tests/cases/conformance/parser/ecmascript5/TupleTypes/TupleType5.ts(1,15): error TS1009: Trailing comma not allowed.
-
-
-==== tests/cases/conformance/parser/ecmascript5/TupleTypes/TupleType5.ts (1 errors) ====
-    var v: [number,]

```

Comment:
You also need to handle `[number,,]`

Reply:
@sheetalkamat ok, added additional tests